extensions/cornerstone/src/init.tsx

```diff
@@ -77,18 +77,18 @@ export default async function init({
 
   window.services = servicesManager.services;
 
-  if (!window.crossOriginIsolated) {
-    uiNotificationService.show({
-      title: 'Cross Origin Isolation',
-      message:
-        'Cross Origin Isolation is not enabled, volume rendering will not work (e.g., MPR)',
-      type: 'warning',
-    });
-  }
-
-  if (cornerstone.getShouldUseCPURendering()) {
-    _showCPURenderingModal(uiModalService, hangingProtocolService);
-  }
+  // if (!window.crossOriginIsolated) {
```

Comment:
This seems like something which could be turned off in the config for the cornerstone extension, as part of the overall configuration file.  Then, the testing can turn it off, but otherwise we can leave it on because it causes issues.